hamilton/registry.py

```diff
@@ -76,15 +76,7 @@ def load_extension(plugin_module: str):
 
     :param plugin_module: the module name sans .py. e.g. pandas, polars, pyspark_pandas.
     """
-    mod = importlib.import_module(f"hamilton.plugins.{plugin_module}_extensions")
-    assert hasattr(mod, "register_types"), "Error extension missing function register_types()"
-    assert hasattr(
-        mod, f"get_column_{plugin_module}"
-    ), f"Error extension missing get_column_{plugin_module}"
-    assert hasattr(
-        mod, f"fill_with_scalar_{plugin_module}"
-    ), f"Error extension missing fill_with_scalar_{plugin_module}"
-    logger.info(f"Detected {plugin_module} and successfully loaded Hamilton extensions.")
```

Comment:
can we not make this conditional?
e.g. have the module declare some variable, and if that's not present we do these asserts?

Reply:
Sure, that works